amd_openvx/openvx/ago/ago_drama_alloc.cpp

```diff
@@ -274,6 +285,23 @@ int agoGpuOclAllocBuffers(AgoGraph * graph)
 				Gd[j][i]->opencl_buffer_offset = Gd[j][k]->opencl_buffer_offset;
 			}
 		}
+#else
+        if (agoGpuHipAllocBuffer(Gd[j][k]) < 0) {
+            return -1;
+        }
+        for (size_t i = 0; i < Gd[j].size(); i++) {
+            if(i != k) {
+                if(Gd[j][i]->alias_offset > 0) {
+                    Gd[j][i]->hip_memory = Gd[j][k]->hip_memory + Gd[j][i]->alias_offset;
+                    Gd[j][i]->hip_memory_allocated = Gd[j][k]->hip_memory;
+                }
+                else {
+                    Gd[j][i]->hip_memory = Gd[j][k]->hip_memory;
+                }
+                Gd[j][i]->opencl_buffer_offset = Gd[j][k]->opencl_buffer_offset;
```

Comment:
We should change the name to gpu_buffer_offset, since it is used for both OpenCL and HIP backend

Reply:
Next PR